evdev/src/Evdev/Codes.chs

```diff
@@ -617,7 +617,8 @@ module Evdev.Codes where
     ABS_TOOL_WIDTH as AbsToolWidth,
     ABS_VOLUME as AbsVolume,
     ABS_MISC as AbsMisc,
-    ABS_RESERVED as AbsReserved,
+    -- Was accidentally removed in 4.19: https://patchwork.kernel.org/patch/10715015/
+    0x2e as AbsReserved,
```

Comment:
Is this actively causing an issue for you? Thanks for bringing it to my attention, but now that I think about it, I'd generally rather not complicate things by implementing workarounds for long-since-fixed upstream bugs, without good reason (eg. one of the broken kernel versions being the basis for a particular widely-used LTS distro release).

Reply:
nah it shouldn't be a problem anymore, we have the evdev headers from linux 5.3, it does have `ABS_RESERVED`